src/reference/sectionB_elements.inc

```diff
@@ -298,14 +298,14 @@ A :code:`reset` element information item (referred to in this specification as a
 
    2. A :code:`reset` element MUST contain exactly two element children, which MUST be one each of the following types:
 
-      .. container:: issue-reset-test-value
-
-         1. A :code:`test_value` element; and,
-
       .. container:: issue-reset-reset-value
 
          2. A :code:`reset_value` element.
 
+      .. container:: issue-reset-test-value
+
+         1. A :code:`test_value` element; and,
```

Comment:
need to change the numbers here?

Reply:
and the punctuation/and ...

Reply:
Yes please!

Reply:
@kerimoyle could you make these changes?

Reply:
Done in commit f5081f9.